### DISCUSSION TITLE
Configurable max # of sensors active?

### DISCUSSION THREAD

**Discussion by chrismartino** [isalkind/homebridge-away-mode#31] [Q&A] [answered]
Hi! Is it possible to configure the max number of sensors enabled at any time?

For example, I have a total of 7 sensors configured, each containing a single light. I don't want all 7 sensors (aka lights) to be on at the same time, so is it possible to configure something like "up to 3 sensors can be active at a time"?

Thanks!

**Comment by isalkind** [comment, marked as answer]
Hey, @chrismartino. It is not currently possible. However, I think it would be a great addition to the feature set and will be adding it in the next couple of days or so. I just need a chance to finalize the code, do some testing, and update the documentation.

**Reply by chrismartino** [reply]
Awesome! Thanks!

**Reply by isalkind** [reply]
This functionality is now available in version `1.6.6`.

Configure by either:
* `maxOnSensors` in config.json, or
* `Global Settings / Miscellaneous / Maximum number of sensors on at a time` in config-ui-x.

**Reply by chrismartino** [reply]
Awesome! Thank you @isalkind!
